cfpb/consumerfinance.gov
# Configures Synonyms to load directly into indexes

Reads synonyms for english and spanish into our synonym analyzer

---


## Additions

- Adds support for spanish synonyms
- Configures local, test, and deployed synonym paths.


## Changes

- How we load synonyms.


## How to test this PR

1. Rebuild your ES7 indexes, confirm the settings for indexes that use synonyms have all synonyms loaded.
  * This can be done via `manage.py search_index --rebuild --parallel -f`



## Checklist



  - PR titles are used to generate the change log in [releases](../../releases); good ones make that easier to scan.
  - Consider prefixing, e.g., "Mega Menu: fix layout bug", or "Docs: Update Docker installation instructions".
  - [This repo’s docs](https://cfpb.github.io/consumerfinance.gov/) (edit the files in the `/docs` folder) – for basic, close-to-the-code docs on working with this repo
  - CFGOV/platform wiki on GHE – for internal CFPB developer guidance
  - CFPB/hubcap wiki on GHE – for internal CFPB design and content guidance
